<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1393</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="titleBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>⚒️ 작업 통계 ⚒️</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>◀️    Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>로그아웃</string>
   </property>
  </widget>
  <widget class="QLabel" name="idBack">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(85, 85, 127, 25); border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="progressBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>611</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(85, 85, 127, 25); border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statisticBack">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>731</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(85, 85, 127, 25); border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="workTable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>115</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>작업명</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>작업자 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>작업 날짜 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>작업 상태 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>후아유 강아지 조명 </string>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>0001</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>ㅇㅇㅇ</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>20240322</string>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>완료</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>후아유 테이블</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>0003</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>ㅁㅁㅁ</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>20240322</string>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string>불량</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>5</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>장난감 </string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>0002</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>ㄱㄴㄷ</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>20240323</string>
    </property>
   </item>
   <item row="2" column="4">
    <property name="text">
     <string>완료</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="errorButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>610</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>⚒️ 불량 보고서 확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="taskStatisticButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>⚒️ 작업별 통계 확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="workerStatisticButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>⚒️ 작업자별 통계 확인</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>701</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="taskgraphLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>590</y>
     <width>701</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="workergraphLayout"/>
  </widget>
  <widget class="QLabel" name="workTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color: rgb(110, 110,130); border-radius: 20px</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;⚒&lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt; 작업 통계 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="ID">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Light</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>ID      :  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Name">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto Light</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>작업자   :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="errorTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>591</width>
     <height>191</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>115</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>작업명</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>작업자 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>작업 날짜 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>불량 단계</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>불량 사유</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="workCombobox">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>후아유 강아지 조명</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>후아유 책상</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>후아유 장난감</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>후아유 테이블</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>후아유 의자</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>기타</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);color: rgb(110, 110,150); border-radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="personCombobox">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>530</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <zorder>idBack</zorder>
  <zorder>progressBack</zorder>
  <zorder>titleBox</zorder>
  <zorder>backButton</zorder>
  <zorder>logoutButton</zorder>
  <zorder>statisticBack</zorder>
  <zorder>workTable</zorder>
  <zorder>errorButton</zorder>
  <zorder>taskStatisticButton</zorder>
  <zorder>workerStatisticButton</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>workTitle</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>errorTable</zorder>
  <zorder>workCombobox</zorder>
  <zorder>label</zorder>
  <zorder>personCombobox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
